src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -552,6 +551,67 @@ public void createSnapshotRowIdsParquetFile(
     executeSynapseQuery(sqlCreateRowIdTable.render());
   }
 
+  public Map<String, Long> createSnapshotParquetFilesByQuery(
+      AssetSpecification assetSpec,
+      UUID snapshotId,
+      String datasetDataSourceName,
+      String snapshotDataSourceName,
+      String translatedQuery,
+      Boolean isGlobalFieldIds)
```

Comment:
If this field was `null`, it would trigger an NPE when `generateSnapshotParquetCreateQuery()` is called. So it should be declared `boolean` instead of `Boolean` here and everywhere else in this class.
```suggestion
      boolean isGlobalFieldIds)
```